src/panels/logbook/ha-logbook.ts

```diff
@@ -117,7 +117,10 @@ class HaLogbook extends LitElement {
       : undefined;
     const item_username =
       item.context_user_id && this.userIdToName[item.context_user_id];
-    const domain = item.entity_id ? computeDomain(item.entity_id) : item.domain;
+    const domain = item.entity_id
+      ? computeDomain(item.entity_id)
+      : // Domain is there if there is no entity ID.
```

Comment:
I changed the type on logbook entry based on the data that I saw. I think that we need to expand the logbook entry type and make it a union type in the future.